requirements/audio.txt

```diff
@@ -25,4 +25,5 @@ torchaudio
 tqdm
 ttsfrd>=0.0.3
 unidecode
-wenetruntime
+# wenetruntime version should be the same as torch
+wenetruntime==1.11
```

Comment:
check wenetruntime和pytorch version是否匹配，在运行wenet pipeline前，如果不匹配给出提示安装特定版本的wenetruntime   eg  pip install wenetruntime==xx

Reply:
+1. 

couple of things here:
1. here no wenetruntime version should be specified
2. the runtime check should make sure that if there is any discpretenchy in wenetruntime and pytorch version, instruct the user to uninstall wenetruntime, and reintall wenetruntime==xxx(the correct version)

such nuance will only bother users that try to setup running environment on his/her own dev-box. for the official modelscope docker image, we will make sure the wenetruntime version align with pytorch version, so that user can use wenet out of box.

Reply:
there may not be a perfect solution for conflicting depdendencies... but let me try to explain the numpy case. 

due to the incompatibilities between tf 1.x and 2.x, a large number of tf models are based on 1.x which in terms require numpy<=1.18, therefore we specify the version to serve such purpose. however, if user already has tf 2.x installed (and intend to try their luck with some models), then we will also promote user to install the correct version.

for wenetruntime, my understanding is that (correct me if i am wrong) a specific wenet version is bound to a specific python version, therefore specifying a particular wenetruntime version may not be particularly useful since a different torch version may break the assumption. rather, it may not be more confusing than helping.

as to the second question,  i assume the wenentruntime version has a 1-to-1 mapping to the torch version? in such case, "the runtime check should make sure that if there is any discpretenchy in wenetruntime and pytorch version, instruct the user to uninstall wenetruntime, and reintall wenetruntime==xxx(the correct version)" here the xxx can be the detected torch version.